matrixify.py

```diff
@@ -13,25 +15,41 @@
 handler.setFormatter(logging.Formatter('%(asctime)s - %(name)s - %(levelname)s - %(message)s'))
 logger.addHandler(handler)
 
-LANGS = {
-    "python",
-    "javascript",
-    "java",
-    "go",
-}
+class ArchList(list):
+    """
+    A list with a .get method that works like dict.get.
+    It's also very ancient and has dark magical powers.
```

Comment:
Ahaha, this is amazing

Time for a patch to the Python stdlib?

Reply:
🙃